setup/apache.yml

```diff
@@ -18,7 +18,7 @@ apache_cosign_deny_friend:     "global=app_cosign_deny_friend default=no"
 
 apache_static_path:
   global: deploy_root
-  output: File.join(deploy_root, app_name, "shared", "public")
+  output: File.join(deploy_root, app_name, "current", "public")
```

Comment:
Symlinking all of public seems reasonable especially since all the precompiled assets are already being served out of shared/public anyway. I can check with them tomorrow if that's feasible, but that;s probably not a change they're going to want to make this week.